pkg/kubelet/sysctl/safe_sysctls.go

```diff
@@ -25,24 +25,24 @@ import (
 	"k8s.io/kubernetes/pkg/proxy/ipvs"
 )
 
-// refer to https://github.com/torvalds/linux/commit/122ff243f5f104194750ecbc76d5946dd1eec934.
-const ipLocalReservedPortsMinNamespacedKernelVersion = "3.16"
+const (
+	// refer to https://github.com/torvalds/linux/commit/122ff243f5f104194750ecbc76d5946dd1eec934.
+	ipLocalReservedPortsMinNamespacedKernelVersion = "3.16"
+	// refer to https://github.com/torvalds/linux/commit/13b287e8d1cad951634389f85b8c9b816bd3bb1e.
+	tcpKeepAliveTimeMinNamespaceKernelVersion = "4.5"
+)
 
-var safeSysctls = []string{
-	"kernel.shm_rmid_forced",
-	"net.ipv4.ip_local_port_range",
-	"net.ipv4.tcp_syncookies",
-	"net.ipv4.ping_group_range",
-	"net.ipv4.ip_unprivileged_port_start",
-}
+var (
+	sysctlIpLocalReserverdPorts = "net.ipv4.ip_local_reserved_ports"
+	sysctlTcpKeepAliveTime      = "net.ipv4.tcp_keepalive_time"
```

Comment:
typo ("Reserved" has an extra "r" at the end)

but also, "Ip" and "Tcp" should be "IP", "TCP"; golang capitalization convention is that if the first letter of an acronym is capitalized, then the whole acronym should be. (So `tcpKeepAliveTime`, but `sysctlTCPKeepAliveTime`)

Reply:
Nice catch.